cmd/orb.go

```diff
@@ -310,7 +310,8 @@ func createOrb(cmd *cobra.Command, args []string) error {
 
 	response, err := api.CreateOrb(ctx, Logger, args[0], args[1])
 
-	if err != nil {
+	// Only fall back to native graphql errors if there are no in-band errors.
+	if err != nil && response == nil && len(response.Errors) == 0 {
```

Comment:
Same question re: response being nil and response.errors == 0

Reply:
same response as above :)